### DISCUSSION TITLE
Deepin Compressor ask the same password twice when extracting two unrelated files with same password

### DISCUSSION THREAD

**Discussion by eduardomozart** [linuxdeepin/developer-center#7168] [Features Request & Ideas | 特性请求 & 头脑风暴]
When extracting two unrelated files on DDE-File-Manager using Deepin Compressor on Deepin 20.9, Deepin Compressor asks the password twice when attempting to extract the second file after extracting the first one. It would be great if Deepin Compressor attempts to use the same password of the first file before asking the password of the second one (similar to https://github.com/linuxdeepin/deepin-compressor/issues/14).

**Comment by BLumia** [comment]
cc @babyfengfjx
